### PR TITLE
Update background_jobs_configuration.rst

### PR DESCRIPTION
**Pull request by fsamapoor**
Minor fixations.

### CODE REVIEW

File: admin_manual/configuration_server/background_jobs_configuration.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Background jobs
 A system like Nextcloud sometimes requires tasks to be done on a regular basis
 without the need for user interaction or hindering Nextcloud performance. For
 that purpose, as a system administrator, you can define background jobs (for
-example, database clean-ups) which are executed without any need for user
+example, database clean-ups) which are executed without any need for the user
```
Comment (marcelklehr) [inline admin_manual/configuration_server/background_jobs_configuration.rst:7]: I think this is correct the way it was

File: admin_manual/configuration_server/background_jobs_configuration.rst [hunk 2]
```diff
@@ -68,11 +68,11 @@ access your server using the Internet. For example::
 
   URL to call: http[s]://<domain-of-your-server>/nextcloud/cron.php
 
-.. warning:: Since WebCron is still executed via web, the webserver in most case limits the
-   resources on the execution. To avoid interrupts inside jobs only 1 jobs is executed
+.. warning:: Since WebCron is still executed via the web, the webserver in most cases limits the
+   resources on the execution. To avoid interrupting inside jobs only 1 job is executed
```
Comment (marcelklehr) [inline admin_manual/configuration_server/background_jobs_configuration.rst:72]: Not sure on interrupts. Feels correct the way it was.

**Comment by marcelklehr** [comment]
Other than my nitpicks above, this looks good! Thank you!

**Comment by fsamapoor** [comment]
> Other than my nitpicks above, this looks good! Thank you!

Done. Thank you for reviewing the changes.

**Review by marcelklehr — APPROVED** [review]

**Pull request merged**